### ISSUE TITLE
Explicit definition of sigma values for morphological filters

### ISSUE THREAD

**Issue by acitcratna** [fiji/Trainable_Segmentation#50]
**TL;DR:** Feature checkboxes for WEKA attributes aren't enough

**TL;DR (+1):** I can use WEKA to filter and remove features/attributes I don't need for classification, but I can't prevent TWS from generating those features/attributes because I only get to choose a range of sigma values. I have a lot of data, and It's wasting a lot of compute time.

I need more control over sigma values than a range between min/max sigma. I don't fully expect any of the following requests to happen, but I'll never give up hope:

**1)** Please make it possible to explicitly define explicit sigma values as a list of discrete numbers instead of a range (between minimum and maximum).

**2)** Even better, please enable explicit definition of sigma values for each **individual** morphological filter. If I need StdDev values ranging from sigma 1 to 4, I shouldn't have to calculate Difference of Gaussians with sigmas of 1-2, 1-3, 2-3, 3-4... or derivatives at sigmas 1, 2, 3 & 4 when I need it at 8.

Please PLEASE let me define explicit values for sigma

**Comment by ctrueden** [comment]
@acitcratna Thank you for suggestions!

> I don't fully expect any of the following requests to happen, but I'll never give up hope

Since this is open-source software, you can do better than hope: you (or your group/colleagues/etc.) can improve the plugin yourself and file a pull request here on GitHub! It would be warmly received.

@iarganda Is there anyone in your group able to investigating making improvements like this?

@maarzt What is the status of [Labkit](https://imagej.net/Labkit)? Would it address @acitcratna's concerns already?

**Comment by maarzt** [comment]
> @maarzt What is the status of [Labkit](https://imagej.net/Labkit)? Would it address @acitcratna's concerns already?

Yes, Labkit addresses this. It allows to specify sigmas individually for each feature.
